### ISSUE TITLE
Error unboxing chat message

### ISSUE THREAD

**Issue by i90rr** [keybase/client#6226]
Keybase GUI Version: 1.0.20-20170309192151+e30557a

![screen shot 2017-03-11 at 10 36 55 pm](https://cloud.githubusercontent.com/assets/394089/23828268/aa2c8022-06ab-11e7-8b51-63a3bc2af954.png)

How should I proceed?
Cheers.

**Comment by maxtaco** [comment]
Cc: @oconnor663 and @mlsteele !

**Comment by eif0** [comment]
Just to add more info to the bug, it's happening not only in the GUI but also in the CLI
```▶ ERROR GetConversationForCLILocal error: error unboxing chat message: chat message body hash replay detected, 000013eea22849cb4487e4f8590df399b6f7d550692883a0f0ee759236ef8a36:40 != 000013eea22849cb4487e4f8590df399b6f7d550692883a0f0ee759236ef8a36:39
```

**Comment by oconnor663** [comment]
The fix for this has landed, and it will go our in one of our next releases.

**Comment by i90rr** [comment]
Great news @oconnor663; out of curiosity: what was the issue?

**Comment by oconnor663** [comment]
The underlying issue is that sometimes our clients send the same message twice, probably when they fail to receive the server's reply for some reason, even though the first send was successful. (We're going to need to fix that on the server with a separate PR.) As it turns out, these double sends end up looking like "replay attacks" to the recipients, and recipients reject them, as part of checking that an Evil Keybase Server isn't messing with conversation history. The UI error in this issue was that the replay error was fatal to the entire conversation, and we needed to recategorize it to be fatal just to the one message.

**Comment by i90rr** [comment]
Jack,
It make sense then 🤔
Before reporting this problem I was to open a new issue informing you guys how unstable behaves the Keybase client when connected to poor internet connections:
A couple of days ago I connected to the internet using my phone LTE plan; it turned out that the connection was quite unstable - with  _lots_ of micro disconnections/reconnections - and it was at that time when I stumbled with the reported issue.
I hope this extra information be of help, let me know if you would prefer I put this on a new ticket.
Cheers.
